resources/frameworks.yaml

```diff
@@ -94,3 +94,11 @@ GAMA:
 autoxgboost:
   version: '0.0.0.9000'
   project: https://github.com/ja-thomas/autoxgboost
+
+supervised:
+  version: '0.3.4'
+  project: https://github.com/mljar/mljar-supervised
+#  params:
+#    explain_level: 0
```

Comment:
given your comment regarding explanation, you can reasonable disable this feature here and explain why:
```
params:
   explain_level: 0   #unnecessary for benchmarks and time-consuming

Reply:
OK!